src/main/java/it/gov/pagopa/pu/workflow/wf/paymentsreporting/broker/config/OrganizationsBrokeredRetrieveWFConfig.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.workflow.wf.paymentsreporting.broker.config;
+
+import io.temporal.workflow.Workflow;
+import it.gov.pagopa.payhub.activities.activity.organization.BrokersRetrieverActivity;
+import it.gov.pagopa.payhub.activities.activity.organization.OrganizationBrokeredRetrieverActivity;
+import it.gov.pagopa.pu.workflow.config.BaseWfConfig;
+import it.gov.pagopa.pu.workflow.config.TemporalWFImplementationCustomizer;
+import it.gov.pagopa.pu.workflow.wf.paymentsreporting.broker.wfretrieve.OrganizationsBrokeredRetrieveWFImpl;
+import it.gov.pagopa.pu.workflow.wf.paymentsreporting.broker.activity.ChainOrganizationsBrokered2OrganizationPaymentsReportingActivity;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "workflow.organizations-brokered")
```

Comment:
this name is ambigous...
you are retrieving brokered organizations, but in order to fetch their paymentsReporting from PagoPa
```suggestion
@ConfigurationProperties(prefix = "workflow.payments-reporting-pagopa-brokers-fetch")
```

ps: please, fix the class
`it.gov.pagopa.pu.workflow.wf.pagopa.paymentreporting.config.OrganizationPaymentsReportingPagoPaFetchWfConfig`
I see a reference to a property, which is not present on the application.yml!
fix also the property name in: `payments-reporting-pagopa-fetch`

use same package of it.gov.pagopa.pu.workflow.wf.pagopa.paymentreporting (see on debtPosition what have been done in order to host more wf on the same package (let's mantain separate Config class here)
(let's call the wf package wfbrokersfetch